test/api/flat-dedupes.test.js

```diff
@@ -1,56 +1,72 @@
-import { Generator } from '@jspm/generator';
-import { strictEqual } from 'assert';
+import { Generator } from "@jspm/generator";
+import { strictEqual } from "assert";
 
 const BASE_CONFIG = {
-  mapUrl: 'about:blank',
+  mapUrl: "about:blank",
   ignore: [
-    'react',
-    'react/jsx-runtime',
-    'react-dom',
-    'react-dom/server',
-    'framer',
-    'framer-motion',
-    'framer-motion/three',
+    "react",
+    "react/jsx-runtime",
+    "react-dom",
+    "react-dom/server",
+    "framer",
+    "framer-motion",
+    "framer-motion/three",
   ],
-  env: ['production', 'browser', 'module'],
+  env: ["production", "browser", "module"],
+  resolutions: {
+    three: "0.142.0",
+    zustand: "3.7.1",
+    ethers: "5.7.2", // workaround for incorrect version pin in @wagmi/core
```

Comment:
Turns out it wasn't a bug in the generator, we just only included `resolutions` with `generatorOne`, which doesn't install `ethers` 🙄 the other generators were correctly resolving `ethers` to latest, since they didn't have explicit resolutions and `ethers` isn't pinned or installed beforehand.

Reply:
Nice.